src/styles/StyleUtils.js

```diff
@@ -1161,6 +1186,16 @@ function getMentionTextColor(isOurMention) {
     return isOurMention ? themeColors.ourMentionText : themeColors.mentionText;
 }
 
+/**
+ * Returns style object for the mobile on WEB
+ * @param {Number} windowHeight
+ * @param {Number} viewportOffsetTop
+ * @returns {Object}
+ */
+function getOuterModalStyle(windowHeight, viewportOffsetTop) {
+    return Browser.isMobile() && {maxHeight: windowHeight, marginTop: viewportOffsetTop};
```

Comment:
@perunt Just to be safe, I think it's better to return empty objects on a non-mobile browser.
```suggestion
    return Browser.isMobile() ? {maxHeight: windowHeight, marginTop: viewportOffsetTop} : {};
```

Reply:
It makes sense, thanks!